typings/web-inspector.d.ts

```diff
@@ -61,17 +66,7 @@ declare global {
       securityOrigin(): string;
     }
 
-    export interface ResourceType {
-      _category: ResourceCategory;
-      name(): string;
-      _name: string;
-      title(): string;
-      isTextType(): boolean;
-    }
-
-    export interface ResourceCategory {
-      title: string;
-    }
+    export type ResourceType = _ResourceType;
```

Comment:
it doesn't seem like anything is referring to this type except the `NetworkRequest` interface, so can drop this declaration?

Reply:
done